proto/buf.gen.yaml

```diff
@@ -4,6 +4,7 @@ managed:
   go_package_prefix:
     default: github.com/bufbuild/modules/private/gen
 plugins:
-  - plugin: buf.build/protocolbuffers/go
+  - plugin: go
+    path: .tmp/bin/protoc-gen-go
```

Comment:
this is just from the Makefile's default `BIN`, right?

Reply:
Yep. We were installing it but not referencing it anywhere.